_data/simple-icons.json

```
@@ -9615,6 +9615,11 @@
             "hex": "37BEFF",
             "source": "https://roundcube.net/"
         },
+        {
+            "title": "rsocket",
+            "hex": "EF0092",
+            "source": "https://rsocket.io/"
```

```suggestion
            "source": "https://github.com/rsocket/rsocket-website/blob/7c97e27c313a3ae5a9356fd4be8044ea72714568/static/img/r-socket.svg"
```

Use this link instead.